### PR TITLE
Merge send_string functions in python debugger client

### PR DESCRIPTION
**Pull request by DanielBallaSZTE**
Merging the send_string messages to one check, therefore multiple duplicated lines are removed.
I didn't find a good solution though in the HTML client, since it would be nested switch cases / ifs, which wouldn't look okay IMO, the other solution would only save like 2 lines of code overall.

JerryScript-DCO-1.0-Signed-off-by: Daniel Balla dballa@inf.u-szeged.hu

### CODE REVIEW

File: jerry-debugger/jerry-client-ws.py [hunk 1]
```diff
@@ -1089,33 +1089,46 @@ def main():
...
 
                 data = debugger.get_message(True)
                 buffer_type = ord(data[2])
                 buffer_size = ord(data[1]) - 1
 
-                if buffer_type not in [eval_type,
-                                       eval_type + 1]:
-                    raise Exception("Eval result expected")
+            if buffer_type not in [msg_type, msg_type + 1]:
+                raise Exception("Invalid data caught")
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.py]: This check should preceede the break statement above

**Comment by DanielBallaSZTE** [comment]
@zherczeg Thanks for the review, PR updated.

File: jerry-debugger/jerry-client-ws.py [hunk 2]
```diff
@@ -1089,33 +1089,47 @@ def main():
...
             while True:
-                message += data[3:]
-
-                if buffer_type == JERRY_DEBUGGER_EVAL_RESULT_END:
-                    subtype = ord(message[-1])
-                    message = message[:-1]
-                    break
+                if buffer_type in [JERRY_DEBUGGER_EVAL_RESULT_END,
+                                   JERRY_DEBUGGER_OUTPUT_RESULT_END]:
+                   subtype = ord(data[-1])
```
Comment (LaszloLango) [inline jerry-debugger/jerry-client-ws.py]: wrong indentation

File: jerry-debugger/jerry-client-ws.py [hunk 3]
```diff
@@ -1089,33 +1089,47 @@ def main():
...
+                   message += data[3:-1]
+                   break
+                else:
+                   message += data[3:]
```
Comment (LaszloLango) [inline jerry-debugger/jerry-client-ws.py]: ditto

**Comment by DanielBallaSZTE** [comment]
@LaszloLango Thanks for the review, I fixed them.

**Pull request merged**